### PR TITLE
Add uptime

### PR DESCRIPTION
**Pull request by ahoneybun**
This PR adds uptime output for support to have per JJ on the Happiness Team.

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,9 @@
+system76-driver (20.04.37~~alpha) focal; urgency=low
+
+  * Daily WIP for 20.04.37
```
Comment (crawfxrd) [inline debian/changelog:3]: Need to add an entry for the change.

Comment (ahoneybun) [reply debian/changelog]: Fixed in latest commit.

File: debian/changelog [hunk 2]
```diff
@@ -1,6 +1,6 @@
 system76-driver (20.04.37~~alpha) focal; urgency=low
 
-  * Daily WIP for 20.04.37
+  * Adds an uptime output file for support
```
Comment (crawfxrd) [inline debian/changelog:3]: You need to keep the Daily WIP line, and add a _different_ entry for the change.

Comment (ahoneybun) [reply debian/changelog:4]: Ah sorry about that! Should be good now!

**Review by crawfxrd — APPROVED** [review]

**Review by jacobgkau — APPROVED** [review]
Output is being collected successfully. `uptime` is provided by `procps` which is a dependency of `ubuntu-minimal`, so we can assume that any systems being supported have this command available (although even if it was missing, it shouldn't cause any problems with other functions due to #192.)

**Pull request merged**